src/H5VLconnector.h

```diff
@@ -81,7 +81,9 @@ typedef enum H5VL_dataset_get_t {
 typedef enum H5VL_dataset_specific_t {
     H5VL_DATASET_SET_EXTENT, /* H5Dset_extent                       */
     H5VL_DATASET_FLUSH,      /* H5Dflush                            */
-    H5VL_DATASET_REFRESH     /* H5Drefresh                          */
+    H5VL_DATASET_REFRESH,    /* H5Drefresh                          */
+    H5VL_DATASET_WAIT,       /* H5Dwait                             */
```

Comment:
Dataset wait doesn't belong in 1.12

Reply:
Resolved in [0b97a78](https://github.com/HDFGroup/hdf5/pull/1970/commits/0b97a78647db222e91d0b9a466f5abf8392fb100)

We are now using `H5VL_NATIVE_DATASET_CHUNK_ITER` instead.